src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -546,6 +546,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             getApproverWithQuery,
             approverSelectorContacts,
             getAvatarUrl,
+            feedbackUrl: getFeatureConfig(features, 'activityFeed.tasks').feedbackUrl || '',
```

Comment:
This could use `lodash/get` directly, since that's what `getFeatureConfig` uses under the hood: 

```
feedbackUrl: getProp(features, 'activityFeed.tasks.feedbackUrl', '');
```